qiskit/ignis/measurement/discriminator/__init__.py

```diff
@@ -0,0 +1,37 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+===========================================
+Discriminators (:mod:`qiskit.ignis.measurement.discriminator`)
```

Comment:
I think this doc string should be moved to an `__init__.py` file in the `qiskit.ignis.measurement` directory

```suggestion
Measurement (:mod:`qiskit.ignis.measurement`)
```